testSrc/com/intellij/plugins/haxe/lang/completion/ReferenceCompletionTest.java

```diff
@@ -231,4 +231,12 @@ public void testExtensionMethod1() throws Throwable {
     doTestInclude("ExtensionMethodExt.hx");
   }
   */
+
+  public void testExtensions() throws Throwable {
```

Comment:
In my opinion, it is better to have each test specified and run separately.  When larger tests are built of several smaller tests, as in this case, it makes it harder to see which variation failed.  As it is, when we get a failure, we only know that one of the variations failed, not *which* variation.

You don't have to agree with me or change anything for now.  I am going to take this change as is.  If you decide to submit another PR with variations as separate tests, I will take it.